tests/test_binary_sensor.py

```diff
@@ -132,6 +132,7 @@ async def test_async_update(hass: HomeAssistant, default_sensor):
     await default_sensor.async_update()
     assert default_sensor.available
     assert default_sensor.is_on
+    assert default_sensor.icon == ICON_WINTER
```

Comment:
This is a test for True. OK.

In line 121, you must also insert this check. For consistency and not to confuse those who will write / read the code after us.

But there is no test for False.
It needs to be added. Line 147